documentation/tarron_notes.md

```diff
@@ -2,12 +2,33 @@
 
 ## Questions
 
---
+- [ ] In the following code, it looks like each badge will have a "user" property that will return a user. Since we decided to just have one badge and we'll tie that to all the users that have earned it, should we get rid of this?
+- [ ] Since our Functions code doesn't have access to create/delete users, will all that have to be done on the front end? For the firebase user only, not the firestore user.
+- [ ] Profiles don't have the user id on them, making them fail the "self" check in auth.js
```

Comment:
This is easy - we just have to put the user ID into the profile when it is generated so that it has the user's ID.